code_generation/producers/genparticles.py

```diff
@@ -127,6 +133,18 @@
     output=[q.gen_match_2],
     scopes=["mt", "et", "tt"],
 )
+gen_taujet_pt_1 = Producer(
+    name="gen_taujet_pt_1",
+    call="quantities::tau::matching_genjet_pt({df}, {output}, 1, {input})",
```

Comment:
I think the index should be "0" here
`call="quantities::tau::matching_genjet_pt({df}, {output}, 0, {input})"`
since it's the first particle.

Reply:
yes